packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -462,7 +460,8 @@ export const RemoteServerFactory = {
             locals.requestStart = Date.now()
             locals.afterResponseCalled = false
             locals.responseCaching = {}
-            locals.requestId = _nextRequestId++
+            that._setRequestId(res, req)
```

Comment:
I like how this works with the dev server and remote server. My OCD is telling me that you should move this down a couple lines after these nicely blocked assignments to `locals` :P

Reply:
It is moved.